codalab/apps/web/bundles.py

```diff
@@ -121,7 +121,16 @@ def worksheet(self, uuid, interpreted=False):
                 for item in worksheet_info['items']:
                     if item['mode'] in ['html', 'contents']:
                         # item['name'] in ['stdout', 'stderr']
-                        item['interpreted'] = map(base64.b64decode, item['interpreted'])
+                        if item['interpreted'] is None:
+                            item['interpreted'] = ['MISSING']
+                        else:
+                            item['interpreted'] = map(base64.b64decode, item['interpreted'])
+                    elif item['mode'] == 'table':
```

Comment:
Remove this section, because resolve_interpreted_items already returns 'MISSING', not None for tables.